x-pack/plugins/cases/common/api/cases/user_actions/index.ts

```diff
@@ -5,23 +5,6 @@
  * 2.0.
  */
 
-import * as rt from 'io-ts';
```

Comment:
I needed to move these because I was getting circular reference errors, so I moved them to `response.ts` so they could be shared between here and the `find.ts` file.